extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3314,12 +3331,12 @@ static public function isWikiPrivate( $wikiId ) {
 	 * @return url in city_list
 	 */
 	static public function DBtoUrl( $dbname, $master = false ) {
-		if ( !self::isUsed() ) {
+		if ( !static::isUsed() ) {
 			Wikia::log( __METHOD__, "", "WikiFactory is not used." );
 			return false;
 		}
 
-		$oRow = self::getWikiByDB( $dbname, $master );
+		$oRow = static::getWikiByDB( $dbname, $master );
 
 		return isset( $oRow->city_url ) ? $oRow->city_url : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)